specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/JobPrivateEndpoints.json

```diff
@@ -0,0 +1,337 @@
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "201": {
+            "description": "Successfully created the private endpoint.",
+            "schema": {
+              "$ref": "#/definitions/JobPrivateEndpoint"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Create a private endpoint.": {
+            "$ref": "./examples/CreateOrUpdateJobPrivateEndpoint.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "JobPrivateEndpoints"
+        ],
+        "description": "Deletes a private endpoint.",
+        "operationId": "JobPrivateEndpoints_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "jobAgentName",
+            "in": "path",
+            "description": "The name of the job agent.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "privateEndpointName",
+            "in": "path",
+            "description": "The name of the private endpoint to delete.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
Long-running delete operations must have responses with 202, 204 and default return codes. They also must have no other response codes.

Please remove the 200 status code.

Reply:
Our current generator creates these responses automatically. You can see that other SQL swagger files follow the same standard. 

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/JobAgents.json#L196C3-L211C3